staging/src/k8s.io/apiserver/pkg/util/feature/feature_gate_test.go

```diff
@@ -124,7 +162,7 @@ func TestFeatureGateFlag(t *testing.T) {
 			testAlphaGate: {Default: false, PreRelease: Alpha},
 			testBetaGate:  {Default: false, PreRelease: Beta},
 		})
-		f.AddFlag(fs)
+		f.AddFlag(fs, "")
 
```

Comment:
we also don't seem to be testing new functionality for `KnownFeatures`

Reply:
Thanks. Added.